test/SConscript

```diff
@@ -268,6 +268,33 @@ test_root = '#interfaces/cython/cantera/test'
 for f in mglob(localenv, test_root, '^test_*.py'):
     python_subtests.append(f.name[5:-3])
 
+if localenv['python_package'] != 'none':
+    # We choose ruamel.yaml 0.15.94 as the minimum version,
```

Comment:
0.15.87 ... ?

Reply:
Actually
```
 ruamel_yaml-0.15.80        |py37h8f50634_1001         267 KB  conda-forge
```
on one of the runners.

Reply:
Update: Ubuntu 18.04 uses 0.15.34 [see link](https://packages.ubuntu.com/source/bionic/ruamel.yaml), and I recall that it worked out of the box.

Reply:
Ok, thank you for helping triangulate this!